### ISSUE TITLE
Rancher/Server Services Naming Error and Updates Needed

### ISSUE THREAD

**Issue by MikePadge** [rancher/os#2686]
**RancherOS Version: (ros os version)**
v1.5.0

**Where are you running RancherOS? (docker-machine, AWS, GCE, baremetal, etc.)**
Baremetal 

`sudo ros service list`
Shows rancher-server and rancher-server-stable as services I can enable. 

rancher-server
rancher-server-stable

Enabling rancher-server-stable and starting it up seems to fail. 
`sudo ros service enable rancher-server-stable `
Pulls the docker.io/rancher/server:stable image fine, but turning it up fails.

`sudo ros service up rancher-server-stable`

Error: No such service: rancher-server-stable 

`sudo ros service up rancher-server `- does however start the "rancher-server-stable" service that was just enabled, but its very confusing. You'll see why in a moment. 

`docker ps -a `shows the service started on port 8080 (I didn't see in the docs where I could declare service ports. Could have easily glazed over. Can this be passed as flag?). 

(NOTE: rancher-server-stable service installs v1.6.26)

ros service disable rancher-server
ros service enable rancher-server (this pulls the docker.io/rancher/server:latest) container image just after stable was shutdown with `ros service disable rancher-server`

ros service up rancher-server - starts correct rancher/server:latest I just enabled. 

(NOTE: rancher-server "latest" service installs v1.6.26 of rancher/server as well)

**Comment by niusmallnan** [comment]
I will drop `rancher-server-stable` and `rancher-server` in os-services.
If you want to install Rancher, I recommend you refer to these documents, you can flexibly define many parameters.
For 1.6: https://rancher.com/docs/rancher/v1.6/en/installing-rancher/installing-server/
For 2.x: https://rancher.com/docs/rancher/v2.x/en/installation/

**Comment by niusmallnan** [comment]
> Error: No such service: rancher-server-stable

I have made a PR to fix it, and backported to v1.5.0/v1.5.1 .
```
[root@ip-172-31-12-6 cache]# rm -rf  /var/lib/rancher/cache/*
[root@ip-172-31-12-6 cache]# ros s enable rancher-server-stable
[root@ip-172-31-12-6 cache]# ros s up rancher-server-stable
INFO[0000] Project [os]: Starting project
INFO[0000] [0/20] [rancher-server-stable]: Starting
INFO[0001] [1/20] [rancher-server-stable]: Started
```
